src/tests/JIT/Regression/JitBlue/Runtime_75832/Runtime_75832.cs

```diff
@@ -8,30 +8,11 @@
 public class Runtime_75832
 {
     [Fact]
-    public static int TestEntryPoint()
-    {
-        try
-        {
-            Test(0);
-            Console.WriteLine("FAIL: No exception thrown");
-        }
-        catch (DivideByZeroException)
-        {
-            return 100;
-        }
-        catch (Exception ex)
-        {
-            Console.WriteLine("FAIL: Caught {0}", ex.GetType().Name);
-        }
-        
-        return 101;
-    }
+    public static void TestEntryPoint()
+        => Assert.Throws<DivideByZeroException>(() => Test(0));
```

Comment:
The general problem I have with these assert helpers is that they expand into a ton of unrelated code that I prefer not to see when I'm looking at the test. I stopped using these generally in JIT tests when I had to skip past tons of inlined `Assert.Equals` gunk while investigating my test.

It's probably ok for this test since most interesting stuff goes on inside `Test`.

Reply:
Thanks for the feedback. This is interesting as I've had the opposite experience when looking at the source code (rather than debugging them) in that there is so much boilerplate in the way. That said, I don't plan on widespread changes to call Equals a bunch but just happened to notice how much boilerplate was needed for Throws while auditing recent test changes. I also came to the conclusion that Throws was "less" than Equals and friends.

It would be nice to have a helper that addresses both sides. I wonder if the xunit folks would accept NoInlining everywhere. Alternatively, we could build our own wrapper library, though it would be unfortunate to end up "not standard".

(Kind of related - [Theory] often ends up not saving much over a wrapper [Fact] method since you often end up needing to pass the expected output and explicitly check against it anyway. It would be nice to have the expected return in the [InlineData]/etc., but again it wouldn't be standard.

Reply:
It's also interesting how often tests define their own AssertEquals, etc.!